### PR TITLE
JDG-2263 Server data volume claim size should be configurable via tem…

### PR DESCRIPTION
**Pull request by ryanemerson**
…plate param in data grid service

https://issues.jboss.org/browse/JDG-2263

### CODE REVIEW

File: services/datagrid-service-template.yaml [hunk 1]
```diff
@@ -214,6 +214,10 @@ parameters:
   displayName: Total Memory
   name: TOTAL_CONTAINER_MEM
   value: "512"
+- description: Sets the total storage capacity, in GiB, available to the application.
```
Comment (galderz) [inline services/datagrid-service-template.yaml]: Wondering if we should specify that this is persistent storage? Users might wonder what's the difference between container memory and storage capacity?

**Review by galderz — COMMENTED** [review]
Just a small comment about the storage capacity description. The rest is fine.

File: services/datagrid-service-template.yaml (hunk 1 above)
Comment (ryanemerson) [reply services/datagrid-service-template.yaml]: Sure, let me know if you like the change and I'll squash

**Review by galderz — DISMISSED** [review]

**Comment by galderz** [comment]
Can you squash commits?

**Comment by ryanemerson** [comment]
@galderz Done

**Review by galderz — APPROVED** [review]

**Pull request merged**